feat: add zip artifacts guide (#528)
screwdriver-cd/screwdriver-cd.github.io: docs/user-guide/environment-variables.md

| Name | Default Value | Description |
|------|---------------|-------------|

> | SD_ZIP_ARTIFACTS | false | **Options:** (`true`/`false`) <br><br>Compresses and uploads artifacts in a single ZIP file.<br><br>**Use case:** Reduces upload time when your build has a lot of artifacts. If the upload fails, it's likely that the zip is larger than can handle.<br><br>**Note:** Consult with your cluster admin to see if this option is available. |

Reviewer: ```suggestion
| SD_ZIP_ARTIFACTS | false | **Options:** (`true`/`false`) <br><br>Compresses and uploads artifacts in a single ZIP file.<br><br>**Use case:** Reduces upload time when your build has a lot of artifacts. If the upload fails, it's likely that the zip is too larger to handle.<br><br>**Note:** Consult with your cluster admin to see if this option is available. |
```
